### PR TITLE
+fluentci.io

### PR DESCRIPTION
**Pull request by tsirysndr**
[fluentci.io](https://fluentci.io) is a CI/CD tool based on [Dagger](https://dagger.io/) and [Deno](https://deno.com)

### CODE REVIEW

File: projects/fluentci.io/package.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+build:
+  dependencies:
+    deno.land: ^1.37
+  working-directory: ${{prefix}}/bin
+  script: |
+    deno compile -A --unstable --import-map="$SRCROOT"/import_map.json --output fluentci "$SRCROOT"/main.ts
+    mkdir -p "{{ prefix }}"/bin
+    cp fluentci "{{ prefix }}"/bin
+  # ^^^^ yes, this is a hack, but it's the only way to get the test to run, and it works
+  # the test fail if we don't copy the binary to the prefix, I don't know why
```
Comment (jhheider) [inline projects/fluentci.io/package.yml]: what about dropping the working-directory and just doing:
```yaml
  script:
    - deno compile -A --unstable --import-map=import_map.json --output fluentci main.ts
    - mkdir -p "{{ prefix }}"/bin
    - install fluentci "{{ prefix }}"/bin
```

File: projects/fluentci.io/package.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  url: https://github.com/fluentci-io/fluentci/archive/refs/tags/v{{ version }}.tar.gz
+  strip-components: 1
+
+display-name: fluentci
+
+versions:
+  github: fluentci-io/fluentci
+
+dependencies:
+  dagger.io: ^0.8.8
```
Comment (jhheider) [inline projects/fluentci.io/package.yml:11]: does it have a runtime dep on dagger? but not deno?

File: projects/fluentci.io/package.yml (hunk 1 above)
Comment (tsirysndr) [reply projects/fluentci.io/package.yml]: I think I've found the problem, we also need to skip this package [here ](https://github.com/pkgxdev/brewkit/blob/a9c140eec89c8a9db429c5c2ffb2c2b03198bbf3/share/brewkit/fix-elf.ts#L26)

File: projects/fluentci.io/package.yml (hunk 2 above)
Comment (tsirysndr) [reply projects/fluentci.io/package.yml:11]: I just added, Deno was missing
I got this message: `pkgx: cyclic dep: deno.land: fluentci.io`, isn't that a problem?

Comment (jhheider) [reply projects/fluentci.io/package.yml:11]: Well, it doesn't need the build dep with the runtime dep. I assume that's why it thinks the dep is cyclic. But a cyclic dep isn't a game stopper, just requires bootstrapping.

File: projects/fluentci.io/package.yml (hunk 1 above)
Comment (tsirysndr) [reply projects/fluentci.io/package.yml]: I created a PR for this : https://github.com/pkgxdev/brewkit/pull/189

**Comment by jhheider** [comment]
finally got it!

**Pull request merged**